packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -21,10 +21,10 @@ import { usePreference } from 'compass-preferences-model/provider';
 import { lenientlyFixQuery } from '../query/leniently-fix-query';
 import type { RootState } from '../stores/query-bar-store';
 import { useAutocompleteFields } from '@mongodb-js/compass-field-store';
-import type { RecentQuery } from '@mongodb-js/my-queries-storage';
 import { applyFromHistory } from '../stores/query-bar-reducer';
 import { getQueryAttributes } from '../utils';
 import type { BaseQuery } from '../constants/query-properties';
+import { SavedQuery } from '@mongodb-js/compass-editor/dist/codemirror/query-history-autocompleter';
```

Comment:
Looks like this import is using the `dist` folder. Let's use the generic package import and make sure we export it there. It also looks like we're only using this as a type so we can use the `type` import syntax. 
```suggestion
import type { SavedQuery } from '@mongodb-js/compass-editor';
```

Since the `SavedQuery` type in `compass-editor` is only used in auto complete how do you feel about a more descriptive name so down the line folks don't get it mixed up? Maybe `SavedQueryAutocompletion`?